extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -5,14 +5,14 @@ public function featured() {
 		$wg = $this->getApp()->wg;
 		$title = $wg->Title->getPrefixedDBkey();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)